cpu/stm32f3/vectors.c

```diff
@@ -27,350 +26,244 @@ void dummy_handler(void) {
+    [80] = isr_tim20_cc,             /* [80] TIM20 Capture Compare Interrupt */
+    [84] = isr_spi4,                 /* [84] SPI4 global Interrupt */
+#elif defined(CPU_MODEL_STM32F303VC)
+    [18] = isr_adc1_2,               /* [18] ADC1 & ADC2 Interrupts */
+    [19] = isr_usb_hp_can_tx,        /* [19] USB Device High Priority or CAN TX Interrupts */
+    [20] = isr_usb_lp_can_rx0,       /* [20] USB Device Low Priority or CAN RX0 Interrupts */
+    [29] = isr_tim3,                 /* [29] TIM3 global Interrupt */
+    [30] = isr_tim4,                 /* [30] TIM4 global Interrupt */
+    [33] = isr_i2c2_ev,              /* [33] I2C2 Event Interrupt & EXTI Line24 Interrupt (I2C2 wakeup) */
+    [34] = isr_i2c2_er,              /* [34] I2C2 Error Interrupt */
+    [35] = isr_spi1,                 /* [35] SPI1 global Interrupt */
+    [36] = isr_spi2,                 /* [36] SPI2 global Interrupt */
+    [42] = isr_usbwakeup,            /* [42] USB Wakeup Interrupt */
+    [43] = isr_tim8_brk,             /* [43] TIM8 Break Interrupt */
+    [44] = isr_tim8_up,              /* [44] TIM8 Update Interrupt */
+    [45] = isr_tim8_trg_com,         /* [45] TIM8 Trigger and Commutation Interrupt */
+    [46] = isr_tim8_cc,              /* [46] TIM8 Capture Compare Interrupt */
+    [47] = isr_adc3,                 /* [47] ADC3 global Interrupt */
+    [51] = isr_spi3,                 /* [51] SPI3 global Interrupt */
+    [52] = isr_uart4,                /* [52] UART4 global Interrupt & EXTI Line34 Interrupt (UART4 wakeup) */
+    [53] = isr_uart5,                /* [53] UART5 global Interrupt & EXTI Line35 Interrupt (UART5 wakeup) */
+    [54] = isr_tim6_dac,             /* [54] TIM6 global and DAC channel 1&2 underrun error  interrupts */
+    [55] = isr_tim7,                 /* [55] TIM7 global Interrupt */
+    [56] = isr_dma2_channel1,        /* [56] DMA2 Channel 1 global Interrupt */
+    [57] = isr_dma2_channel2,        /* [57] DMA2 Channel 2 global Interrupt */
+    [58] = isr_dma2_channel3,        /* [58] DMA2 Channel 3 global Interrupt */
+    [59] = isr_dma2_channel4,        /* [59] DMA2 Channel 4 global Interrupt */
+    [60] = isr_dma2_channel5,        /* [60] DMA2 Channel 5 global Interrupt */
+    [61] = isr_adc4,                 /* [61] ADC4  global Interrupt */
+    [64] = isr_comp1_2_3,            /* [64] COMP1, COMP2 and COMP3 global Interrupt via EXTI Line21, 22 and 29 */
+    [65] = isr_comp4_5_6,            /* [65] COMP4, COMP5 and COMP6 global Interrupt via EXTI Line30, 31 and 32 */
+    [66] = isr_comp7,                /* [66] COMP7 global Interrupt via EXTI Line33 */
+    [74] = isr_usb_hp,               /* [74] USB High Priority global Interrupt remap */
+    [75] = isr_usb_lp,               /* [75] USB Low Priority global Interrupt  remap */
+    [76] = isr_usbwakeup_rmp,        /* [76] USB Wakeup Interrupt remap */
+#elif defined(CPU_MODEL_STM32F334R8)
+    [18] = isr_adc1_2,               /* [18] ADC1 & ADC2 Interrupts */
```

Comment:
This is present is all F3 flavors, I think it could be moved to the common part.